integrations/nvidia/src/haystack_integrations/components/embedders/nvidia/truncate.py

```diff
@@ -0,0 +1,30 @@
+from enum import Enum
+
+
+class TruncateMode(Enum):
```

Comment:
```suggestion
class EmbeddingTruncateMode(Enum):
```
Since we don't use it for the generator.